app/screens/create_direct_message/create_direct_message.tsx

```diff
@@ -1,84 +1,39 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React, {useCallback, useEffect, useMemo, useReducer, useRef, useState} from 'react';
-import {useIntl} from 'react-intl';
-import {Keyboard, Platform, View} from 'react-native';
-import {SafeAreaView} from 'react-native-safe-area-context';
+import React, {useCallback, useState} from 'react';
+import {defineMessages, useIntl} from 'react-intl';
 
 import {makeDirectChannel, makeGroupChannel} from '@actions/remote/channel';
 import {fetchProfiles, fetchProfilesInTeam, searchProfiles} from '@actions/remote/user';
-import CompassIcon from '@components/compass_icon';
-import Loading from '@components/loading';
-import Search from '@components/search';
 import {General} from '@constants';
 import {useServerUrl} from '@context/server';
-import {useTheme} from '@context/theme';
-import {debounce} from '@helpers/api/general';
-import useNavButtonPressed from '@hooks/navigation_button_pressed';
 import {t} from '@i18n';
-import {dismissModal, setButtons} from '@screens/navigation';
+import MembersModal from '@screens/members_modal';
 import {alertErrorWithFallback} from '@utils/draft';
-import {changeOpacity, getKeyboardAppearanceFromTheme, makeStyleSheetFromTheme} from '@utils/theme';
-import {displayUsername, filterProfilesMatchingTerm} from '@utils/user';
-
-import SelectedUsers from './selected_users';
-import UserList from './user_list';
-
-const START_BUTTON = 'start-conversation';
-const CLOSE_BUTTON = 'close-dms';
+import {displayUsername} from '@utils/user';
+
+const messages = defineMessages({
```

Comment:
This looks awesome! Does the i18n-extract script handle this correctly?
If it does, you can remove the `t` function from lines 22 and 26. If it doesn't, you forgot to add the `t` function on line 18.

Reply:
I had tested using this before and found that I had to use the `t` function. Thanks for noticing the missing `t` on line: 18